test/bugs-reported/bug3.chpl

```diff
@@ -0,0 +1,8 @@
+proc max(a:int, b:int): int{
```

Comment:
Created a JIRA Issue for this: https://chapel.atlassian.net/browse/CHAPEL-197

Will remove this as a "future" test.